includes/multi-currency/Settings.php

```diff
@@ -42,7 +35,6 @@ class Settings extends \WC_Settings_Page {
 	 */
 	public function __construct( MultiCurrency $multi_currency ) {
 		$this->multi_currency = $multi_currency;
-		$this->id             = $this->multi_currency->id;
```

Comment:
I think we should keep this ID variable as it is currently, because it looks like the class being extended (`WC_Settings_Page`) uses it in some places, so removing it may have unintended consequences.

Reply:
Good catch! I had only been focusing within MC and didn't look into the WC core class that was being extended. Thank you 😅